Makefile.defs

```diff
@@ -170,6 +170,6 @@ ifeq ($(BASE_IMAGE),)
     BASE_IMAGE=scratch
 endif
 
-HELM_DOCS_VERSION ?= "5ddabba"
-HELM_DOCS_SHA ?= "c4a91daac6bf9c181fac79e09712fcf1f9102fcab1bbacbc733883965f5fd244"
-HELM_DOCS_IMAGE ?= "docker.io/bmcustodio/helm-docs:$(HELM_DOCS_VERSION)@sha256:$(HELM_DOCS_SHA)"
+HELM_DOCS_VERSION ?= "v1.7.0"
```

Comment:
I bumped this in https://github.com/cilium/helm-docs/tree/main, and pushed new `v1.10.0` tags to quay.io. Was there another reason not to follow up on this PR?

Reply:
While we're at it, I also included `helm` in the image so we could replace `helm` direct usage e.g. in `install/kubernetes/Makefile` with instead running this image.

Reply:
LMK if you're interested in following up on that, or otherwise I might pick it up some time when I get a few moments.

Reply:
^ I ended up getting a bit curious and worked on this :sweat_smile: 

https://github.com/cilium/helm-toolbox
https://github.com/cilium/cilium/pull/20236